addons/project/models/project.py

```diff
@@ -1156,6 +1157,8 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
 
     # Project sharing fields
     display_parent_task_button = fields.Boolean(compute='_compute_display_parent_task_button', compute_sudo=True)
+    # Filter fields
+    is_blocked = fields.Boolean(compute='_compute_is_blocked', store=True, recursive=True)
```

Comment:
Can you move this field below the `dependent_tasks_count` field to get all fields related to the task dependencies feature at the same place?
Also you can remove the comment you added since this field could be used in a classic `search` for instance. 

Thanks in advance